test/functionalTest/harnessFunctions.sh

```diff
@@ -278,6 +278,7 @@ function brokerStartAwait
     if [ "$?" == "0" ]
     then
       vMsg The orion context broker has started, listening on port $port
+      echo "Broker started after $loopNo checks" >> /tmp/brokerStart
```

Comment:
This here is the list that 'fixes' the problem.
Hard to believe, right?

Also, even if the echo is done to /dev/null, the 'fix' still works.
Now, at least some info is gathered in /tmp/brokerStart (emptied in test/functionalTest/testHarness.sh)

NTC